lib/shell.js

```diff
@@ -34,20 +34,19 @@ const async = require('async'),
                     }
                 },
                 function(next) {
-                    const execOption = {};
                     if (options.sessionId) {
                         if (options.session.getDevice().username !== 'root') {
                             return setImmediate(next, errHndl.changeErrMsg("NEED_ROOT_PERMISSION", "connect user session"));
                         }
 
                         // -l : make the shell a login shell, -c : command
-                        runCommand = `su ${options.sessionId} -l -c "${runCommand}"`;
-                        execOption.pty = true;
+                        runCommand = `su ${options.sessionId} -l -c '${runCommand}'`;
+                    } else {
+                        runCommand = "source /etc/profile && " + runCommand;
                     }
 
-                    runCommand = "source /etc/profile && " + runCommand;
                     log.info("shell#remoteRun()", "cmd :", runCommand);
-                    options.session.runWithOption(runCommand, execOption, process.stdin, process.stdout, process.stderr, next);
+                    options.session.runWithOption(runCommand, {pty: true}, process.stdin, process.stdout, process.stderr, next);
```

Comment:
@YoungeunKim 
I made the same change as code #109.

Reply:
okay!

Reply:
Could you check that runWithOption can be changed to 
options.session.run(runCommand, process.stdin, process.stdout, process.stderr, next);